content/en/docs/Getting started/deploying-cuebot.md

```diff
@@ -20,9 +20,13 @@ management tasks, including:
 A typical OpenCue deployment runs a single instance of Cuebot, which is shared
 by all users.
 
+
 ## System requirements
 
-A Cuebot server requires a mimimum of 6GB of RAM.
+*   A Cuebot server requires a mimimum of 6GB of RAM.
```

Comment:
As the requirements are now a list, it would be good to add a simple introductory sentence, something like:

To plan your installation of Cuebot, consider the following requirements:

Reply:
Done.